packages/boot/test/bootstrapTests/test-addAssets.ts

```diff
@@ -0,0 +1,196 @@
+import { test as anyTest } from '@agoric/zoe/tools/prepare-test-env-ava.js';
+
+import type { TestFn } from 'ava';
+
+import { TimeMath } from '@agoric/time';
+import {
+  LiquidationTestContext,
+  makeLiquidationTestContext,
+} from './liquidation.ts';
+
+const test = anyTest as TestFn<LiquidationTestContext>;
```

Comment:
grand

Reply:
I copied this from a nearby test. I don't even know what it means. Care to explain? What's it do, and why is it "grand"?

Reply:
grand that you're using TS syntax ;-)

It means: "define `test` to be the value of `anyTest` with the type `TestFn` with its `Context` type parameter set to `LiquidationTestContext`". `TestFn` ensures that any `test` methods and descendants (like `t`) are typed The default `t.context` is `unknown` so the parameter slot fills that in with the actual context being assigned in line 16.

Though I see on line 30 that you're adding `proposal` to the context. To not get a type error, include it here.

```suggestion
const test = anyTest as TestFn<LiquidationTestContext & { proposal: Awaited<ReturnType<typeof buildProposal>> }>;
```

Reply:
Though seeing further down, I don't think the raw proposal should go into the context because it's really more of a template. See request below for providing a helper function instead.

Reply:
done, though the declarations are a little more involved than that.